packages/proxy/test/decrypt.ts

```diff
@@ -0,0 +1,21 @@
+import { decryptMessage } from "../src/encrypt";
```

Comment:
I think we should either remove this file or introduce `vitest`/`jest` into the repo and make this a test. Happy to help with the latter if you're open to it, or I can clean it up in a follow up change.

Reply:
Ok, I removed it. Feel free to add as a test later if you decide to incorporate vitest.